.github/workflows/executeAction.yml

```diff
@@ -28,8 +28,13 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v3
 
+      # https://stackoverflow.com/questions/62627931/github-actions-expression-functions-string-manipulation
+      - name: Set address lowercase
+        id: setAddressLowerCase
+        run: INPUT=${{ fromJSON(inputs.issueInfo).address }} echo "::set-output name=addressLowerCase::${INPUT,,}"
+
```

Comment:
Awesome!

Reply:
Great you added also the stackoverflow ref!